static/js/containers/pages/register/RegisterDetailsPage.js

```diff
@@ -36,6 +42,10 @@ type StateProps = {|
   countries: Array<Country>
 |}
 
+type State = {|
+  authState: ?AuthStates
```

Comment:
Can you add the missing states like `STATE_USER_BLOCKED` and etc to the list of possible values in `AuthStates`?